cluster-http/src/main/resources/reference.conf

```diff
@@ -16,3 +16,65 @@ akka.cluster.http.management {
   # The value will need to be from 0 to 65535.
   port = 19999
 }
+
+######################################################
+# Akka Cluster Bootstrap Config                      #
+######################################################
+
+
+akka cluster.bootstrap {
+
+  contact-point-discovery {
+
+    # define this name to be looked up in service discovery for "neighboring" nodes
+    # if undefined, the name will be extracted from the ActorSystem name
+    service-name = null
```

Comment:
Ok, can do "" and treat it as empty.

Yeah the null allows documeting the key here, but if you check `hasPath` it will return false for the given key. So it's like an option. But you're right, let's just use ""